indico/modules/rb_new/client/js/common/rooms/RoomEditModal.jsx

```diff
@@ -254,35 +260,82 @@ const columns = [
         name: 'notificationsEnabled',
         label: Translate.string('Reminders enabled')
     }, {
-        type: 'input',
-        name: 'notificationBeforeDays',
-        label: Translate.string('Send Booking reminders X days before (single/daily)'),
-        inputArgs: {
-            type: 'number',
-            min: 1,
-            max: 30,
-        },
-        required: false
+        type: 'checkbox',
+        name: 'endNotificationsEnabled',
+        label: Translate.string('Reminders of finishing bookings enabled')
     }, {
-        type: 'input',
-        name: 'notificationBeforeDaysWeekly',
-        label: Translate.string('Send Booking reminders X days before (weekly)'),
-        inputArgs: {
-            type: 'number',
-            min: 1,
-            max: 30,
-        },
-        required: false
+        type: 'header',
+        label: Translate.string('Notifications')
     }, {
-        type: 'input',
-        name: 'notificationBeforeDaysMonthly',
-        label: Translate.string('Send Booking reminders X days before (monthly)'),
-        inputArgs: {
-            type: 'number',
-            min: 1,
-            max: 30,
-        },
-        required: false
+        type: 'formgroup',
+        key: 'notifications',
+        label: Translate.string('How many days in advance booking reminders should be sent?'),
+        content: [{
+            type: 'input',
+            name: 'notificationBeforeDays',
+            label: Translate.string('Single/Daily'),
+            inputArgs: {
+                type: 'number',
+                min: 1,
+                max: 30,
+            },
+            required: false
+        }, {
+            type: 'input',
+            name: 'notificationBeforeDaysWeekly',
+            label: Translate.string('Weekly'),
+            inputArgs: {
+                type: 'number',
+                min: 1,
+                max: 30,
+            },
+            required: false
+        }, {
+            type: 'input',
+            name: 'notificationBeforeDaysMonthly',
+            label: Translate.string('Monthly'),
+            inputArgs: {
+                type: 'number',
+                min: 1,
+                max: 30,
+            },
+            required: false
+        }]
+    }, {
+        type: 'formgroup',
+        key: 'notificationsOfFinishingBookings',
+        label: Translate.string('How many days before the end of a booking reminders should be sent?'),
```

Comment:
s/reminders should/should reminders/ - also in all the other places where you have a `?` at the end. the way it's currently written it'd be fine as an statement but not as a question.